content/en/blog/_posts/2024-04-xx-structured-authz-beta.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: blog
+title: 'Multi-Webhook and Modular Authorization Made Easy'
```

Comment:
```suggestion
title: 'Kubernetes 1.30: Multi-Webhook and Modular Authorization Made Much Easier'
```
?

I'm wary of calling it easy - our readers might disagree, even if the new thing really helps.